build/images/training-operator/Dockerfile.buildx

```diff
@@ -0,0 +1,7 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.3
```

Comment:
should use ubi8 or ubi9?

Reply:
@astefanutti and @anishasthana, what is your take? I think that FIPS compliance is a must.